lib/service/db.go

```diff
@@ -141,6 +145,16 @@ func (process *TeleportProcess) initDatabaseService() (retErr error) {
 		return trace.Wrap(err)
 	}
 
+	// Listen for the TAG database resource if TAG proxying is enabled.
+	resourceMatchers := append(make([]services.ResourceMatcher, 0), process.Config.Databases.ResourceMatchers...)
+	if process.Config.Databases.ProxyTAG {
+		resourceMatchers = append(resourceMatchers, services.ResourceMatcher{
+			Labels: types.Labels{
+				"env": []string{TAGDatabaseEnvLabel},
```

Comment:
This should 100% be a `teleport.internal/foo` or `teleport.dev/foo` label, not `env`.

Reply:
Correction: this should not be here and we should be running a statically configured database (or some equivalent concept that lets us define the TAG database entry at a later time), otherwise any cloud tenant admin will be able to add databases that proxies will pick up.